tokio/src/sync/watch.rs

```diff
@@ -412,18 +418,32 @@ mod state {
         }
 
         /// Load the current value of the state.
-        pub(super) fn load(&self) -> StateSnapshot {
-            StateSnapshot(self.0.load(SeqCst))
+        pub(super) fn load(&self, ordering: Ordering) -> StateSnapshot {
+            StateSnapshot(self.0.load(ordering))
+        }
```

Comment:
This method is used only by the `Debug` formatting. I don't think there's any reason to optimize the `Debug` formatting to use relaxed instead of acquire. I would just delete this method.